### PR TITLE
add ext-authz benchmark files

### PR DESCRIPTION
**Pull request by bochengchu**
Add ext-authz benchmark tests.

### CODE REVIEW

File: perf/benchmark/configs/istio/ext-authz/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Ext-authz Benchmarking
+This directory contains scripts and configurations for ext-authz benchmarking.
+## Prerequisites
+1. See [Istio Performance Benchmarking](https://github.com/istio/tools/tree/master/perf/benchmark) for environment setup.
+2. Ext-authz setup:
```
Comment (aryan16) [inline perf/benchmark/configs/istio/ext-authz/README.md:5]: can you add ext authz setup in your run.sh, let's make it fully automated.

File: perf/benchmark/configs/istio/ext-authz/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+   - Deploy ext-authz in `twopods-istio` namespace
+   - For more, see the [guide](https://istio.io/latest/docs/tasks/security/authorization/authz-custom/).
+
+## Run the tests
+To start the tests,
+```
+./run.sh
+```
+
+## Analysis
```
Comment (aryan16) [inline perf/benchmark/configs/istio/ext-authz/README.md:15]: add the graphs as well

Comment (bochengchu) [reply perf/benchmark/configs/istio/ext-authz/README.md:22]: added the code for generating graphs. 
tools/perf/benchmark/graph_plotter/graph_plotter.py cannot be used for my settings, so I upload my own.

File: perf/benchmark/configs/istio/ext-authz/README.md (hunk 1 above)
Comment (bochengchu) [reply perf/benchmark/configs/istio/ext-authz/README.md:5]: ext-authz setup involves editing a config map, and the environment setup has made the config map too complicated to edit it in script

**Review by aryan16 — APPROVED** [review]

**Review by liwenhao0810 — APPROVED** [review]

**Review by richardwxn — COMMENTED** [review]
why need to use recursive ".." in the jsonpath?

File: perf/benchmark/configs/istio/ext-authz/run.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```
Comment (richardwxn) [inline perf/benchmark/configs/istio/ext-authz/run.sh:1]: try to separate the run.sh into prerun.sh and postrun.sh to align with the framework? check other folder for examples

Comment (bochengchu) [reply perf/benchmark/configs/istio/ext-authz/run.sh:1]: fixed

**Comment by bochengchu** [comment]
> why need to use recursive ".." in the jsonpath?

Changed to use items[0] to align with https://github.com/istio/tools/blob/master/perf/benchmark/runner/runner.py#L54

File: perf/benchmark/configs/istio/ext-authz/run.sh (hunk 3 above)
Comment (richardwxn) [reply perf/benchmark/configs/istio/ext-authz/run.sh:1]: looks like this file has not been renamed to postrun.sh..but others lgtm

**Review by richardwxn — APPROVED** [review]

**Comment by richardwxn** [comment]
@bochengchu you can remove the do-not-merge label after you update the file name

**Comment by bochengchu** [comment]
> @bochengchu you can remove the do-not-merge label after you update the file name

Fixed. @richardwxn could you help me do that? I'm not able to do so.

**Pull request merged**